### DISCUSSION TITLE
Event Json representation exceeds the byte size limit of 262144 set for this Se sink and will be dropped

### DISCUSSION THREAD

**Discussion by mKamiAsim** [datalust/seq-tickets#1508] [Help] [answered]
I want to log about 1.2 MB payload. I had configured Raw event body limit in Seq sever as 10485760. 
I enabled SelfLog in my app and received following error

Event Json representation exceeds the byte size limit of 262144 set for this Se sink and will be dropped.

While logging on File is working fine.

I am using  .Net 5 web api and 
Serilog.AspNetCore 3.4.0
Serilog.Filters.Expressions 2.1.0
Serilog.Settings.Configuration 3.1.0
Serilog.sinks.Seq 5.0.0
Serilog.Sinks.File 4.1.0

**Comment by nblumhardt** [comment, marked as answer]
Hi Muhammad,

When you call `WriteTo.Seq()`, there's another parameter `eventBodyLimitBytes` - you'll also need to pass the bigger value through here, or the sink will drop the event before ever sending it to the server.

Sending 1.2 MB events will impact UI and search performance, however (I assume you have considered this trade-off, but worth extra thought if this is going to a mission critical/production Seq instance).

Hope this helps!
Nick

**Reply by mKamiAsim** [reply]
Thanks @nblumhardt I found configurations in SeqLoggerConfigurationExtensions. Configuring eventBodyLimitBytes worked. 
Regarding performance, we need such payload for monitoring purpose. We ll reconfigure payload size later.
